### ISSUE TITLE
How does TimeDiff generate tags with classes to serve downstream tasks?

### ISSUE THREAD

**Issue by devin-21** [MuhangTian/TimeDiff#4]
Hi Muhang, I noticed in the paper you mentioned that you used synthetic data for usefulness testing, i.e., you trained several classification models under the **TSTR**, **TSRTR** settings.

When I look at the training code and data preprocessing module(i.e. MIMIC III), I don't see how the specific model generates the time series with categories, am I missing something? Can you help me point out how your models generate synthetic data with **class control**? 

Thanks for your excellent work.

**Comment by MuhangTian** [comment]
Hello! Thank you for your interest in our work!

You are right that there are no specific codes to generate labels (in the scenario of healthcare data, the label is whether the patient died or not). In practice, we generate the labels by treating them as a time series. For instance, for a sample with the label of +1, we just have a time series of length $L$ with +1 everywhere.

**Comment by devin-21** [comment]
Thank you for your reply, I saw the code in the sampled code that related to the content of your reply.
Thanks again for your excellent work!
